extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1401,6 +1396,26 @@ export class AmpA4A extends AMP.BaseElement {
   }
 
   /**
+   * Loads the necessary extension scripts, as specified in the metadata.
+   * @param {!Array<!Object<string, string>>} extensionList
+   */
+  loadExtensions_(extensionList) {
+    const extensions = extensionsFor(this.win);
+    extensionList.forEach(
+        extension => {
+          const extensionId = extension['custom-element'];
+          const extensionSrc = extension['src'];
```

Comment:
why do we decide to send src through metadata instead of just the version number?